hudi-aws/src/main/java/org/apache/hudi/aws/sync/AWSGlueCatalogSyncClient.java

```diff
@@ -497,7 +497,11 @@ && getTable(awsGlue, databaseName, tableName).partitionKeys().equals(partitionKe
           .tableInput(updatedTableInput)
           .build();
 
-      awsGlue.updateTable(request);
+      try {
+        awsGlue.updateTable(request).get();
```

Comment:
Does this change impact the access efficiency?

Reply:
The updateTable glue operation is pretty fast. So the impact is negligible.